### DISCUSSION TITLE
Edges are not rendered

### DISCUSSION THREAD

**Discussion by tzelovaka** [xyflow/xyflow#3556] [Q&A]
Hello everyone! My problem: Array of edges comes from express server and rendering process is good. But when array of edges comes from database through server edge renders only after I drag nodes on two ends of edge.
"react-dom.production.min.js:24 Error: <foreignObject> attribute x: Expected length, "NaN"."

**Comment by mikalai-sauchanka** [comment]
This happens to me when onlyRenderVisibleElements is true and one of connecting nodes hasn't been rendered.
